Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md

```diff
@@ -0,0 +1,23 @@
+Displays a geographical location in an indicator field using Google Maps.
```

Comment:
Displays a geographical location using Google Maps.

we are not displaying the location in the field - displays in the indicator layout (that using the `geolocation` field).